apps/website-new/docs/en/guide/basic/runtime.mdx

```diff
@@ -180,6 +196,50 @@ loadShare('react').then((reactFactory) => {
 });
 ```
 
+If has set multiple version shared, `loadShare` will return the loaded and has max version shared. The behavior can be controlled by set `extraOptions.resolver`:
+
+```js
+import { init, loadRemote, loadShare } from '@module-federation/runtime';
+
+init({
+  name: '@demo/main-app',
+  remotes: [],
+  shared: {
+    react: [
+      {
+        version: '17.0.0',
+        scope: 'default',
+        get: async ()=>() => ({ version: '17.0.0)' }),
```

Comment:
```suggestion
        get: async ()=>() => ({ version: '17.0.0' }),
```

Reply:
Thanks for capturing such a detailed issue, modified it!